src/Controllers/TaskController.java

```diff
@@ -0,0 +1,125 @@
+package Controllers;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.UUID;
+
+import DTO.Task;
+import Services.TaskService;
+
+public class TaskController {
+    private static Scanner sc = new Scanner(System.in);
+
+    public static void runTodoApp() {
+        char option = Character.MIN_VALUE;
+
+        System.out.print("TODO APP! Please Select your desired option.\n" +
+                "To add a task. Type a.\n" +
+                "to update a task. Type b.\n" +
+                "To print all tasks. Type c.\n" +
+                "To remove a task. Type d\n");
+
+        option = sc.nextLine().charAt(0);
+
+        performAction(option);
+
+    }
+
+    public static void performAction(char option) {
+        switch (option) {
+            case 'a':
+                addTask();
+                break;
+
+            case 'b':
+                System.out.println("Please enter task Id to update");
+                String taskId = sc.nextLine();
+                updateTask(taskId);
+                break;
+            case 'c':
+                listTasks();
+                break;
+
+            case 'd':
+                System.out.println("Please enter task Id to remove status");
+                String __taskId = sc.nextLine();
+                removeTask(__taskId);
+                break;
+            case 'f':
+                System.out.println("Please enter task Id to update task status");
+                String _taskId = sc.nextLine();
+                updateTaskStatus(_taskId);
+                break;
+            default:
+                System.out.println("Invalid option selected. Please try again!");
+                break;
+        }
+
+        runTodoApp();
+    }
+
+    public static void addTask() {
```

Comment:
Make all these public methods to private as these are only being accessed from this class